classes/middleware/ValidateCsrfToken.php

```diff
@@ -51,7 +51,7 @@ public function handle(Request $request, Closure $next)
      */
     protected function isApiRequest(Request $request): bool
     {
-        return $request->query('apiToken', null) ? true : false;
+        return !!$request->header('authorization');
```

Comment:
This is ok but I think we need to support both case that is as `query param` and `authorization header` . I suggest to update it as 
```php
// API Token may be sent as request query param
if ($request->query('apiToken', null)) {
    return true;
}

// API Token may set as request Authorization header
if ($request->header('authorization', null)) {
    return true;
}

return false;
```